.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,16 @@
 ## Description
-
-<!--- Describe your changes in detail. -->
+<!--- Describe your changes in detail. The sections suggested are intended to make -->
+<!--- it easy to create a descriptive PR that is easy to review. Change as needed! --> 
```

Comment:
+++ I love starting off making it explicit that these are just guidelines, inviting PR authors to change the sections as they see fit.

Can we remove the trailing space on this line?
```suggestion
<!--- it easy to create a descriptive PR that is easy to review. Change as needed! -->
```